sound/hybrid_font.h

```diff
@@ -282,13 +338,16 @@ class HybridFont : public SaberBase {
   bool swinging_ = false;
   void SB_Motion(const Vec3& gyro, bool clear) override {
     float speed = sqrtf(gyro.z * gyro.z + gyro.y * gyro.y);
-    if (speed > 250.0) {
+    if (speed > config_.ProffieOSSwingSpeedThreshold) {
       if (!swinging_ && state_ != STATE_OFF &&
 	  !(lockup.files_found() && SaberBase::Lockup())) {
         swinging_ = true;
-        Play(&swing, &swng);
+        StartSwing();
       }
-    } else {
+      float swing_strength = std::min<float>(1.0, speed / config_.ProffieOSSwingSpeedThreshold);
+      SetSwingVolume(swing_strength, 1.0);
+    } else if (swinging_ && speed <= config_.ProffieOSSwingSpeedThreshold * 
+               ProffieOSSwingLowerThreshold, config_.ProffieOSSwingLowerThreshold) {
```

Comment:
the comma here probably doesn't do what you think.
Besides, it's probably better to just do:

speed <= config_.ProffieOSSwingLowerThreshold and default it to ~200

Reply:
that is simpler, yes.  fixed.